.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Just added .idea/ to .gitignore -- would it be possible to remove the files within the .idea folder from the pull request? (I believe its just some config files) :0